### PR TITLE
FAQ and markdown

### LINKED ISSUE #699: Create FAQ
Similar to https://www.curve.fi/

WIP:
https://docs.google.com/document/d/1p-KUa14y59Ny5BeCD0J0Ft6Nx_Zik0_QGNbDaNQH3Fo/edit

### PR DESCRIPTION
**Pull request by Velenir**
Adds markdown loader (which has some problems with code blocks, but for now we don't have to care)
Adds tenative FAQ.md asset

access at /faq path

Feel free to suggest different styling

As for the text, would be better if headings were real headings not bold text, both in the doc and in md

Closes #699 after a fashion

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr715--dexreact.review.gnosisdev.com

File: src/assets/md/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+**What is Mesa?**
+
+Mesa is a decentralized application built on the **Gnosis Protocol**, a decentralized trading protocol for ERC-20s. Instead of using one order book per token pair, the Gnosis Protocol relies on batched auctions and a multidimensional orderbook. These mechanisms enable ring trades, in which liquidity amongst all assets tradable on the protocol is shared. Read more about the protocol’s mechanism on the Gnosis [DevPortal](https://dfusion-docs.dev.gnosisdev.com/docs/introduction1/).
```
Comment (anxolin) [inline src/assets/md/FAQ.md]: Can we make the link to open in a new window for every link that it's external?

File: src/pages/FAQ.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { ContentPage } from 'components/Layout/PageWrapper'
+import { react as FAQ_CONTENT } from 'assets/md/FAQ.md'
```
Comment (anxolin) [inline src/pages/FAQ.tsx]: Why capitalized? FaqContent would be fine imo

Comment (Velenir) [reply src/pages/FAQ.tsx]: renamed

File: src/assets/md/FAQ.md (hunk 1 above)
Comment (Velenir) [reply src/assets/md/FAQ.md]: Done

**Comment by Velenir** [comment]
> Do we expect the extended page to have this dark background?

**Comment by Velenir** [comment]
@alfetopito 
I don't have your dark footer even in Firefox
You sure you are not accidentally in dark mode? Half-dark mode, footer-only dark mode? Clear localStorage to make sure

**Comment by alfetopito** [comment]
> @alfetopito
> I don't have your dark footer even in Firefox
> You sure you are not accidentally in dark mode? Half-dark mode, footer-only dark mode? Clear localStorage to make sure

I can see it even in private browsing window on FF. Brave and Chrome behave nicely, though.
No dark mode nor anything.

If it's just me, not important then. I can live with that.

**Comment by anxolin** [comment]
I don't see the dark background neither

**Pull request merged**